woocommerce/navigation
# Include `tests/` in PHPCompatibility phpcs rule

Fixes #41 

This PR removes the exclusion of `tests/` from the PHPCompatibility phpcs rule.

### Test instructions

- Check out branch
- Edit a test file, such as `sample-test.php` and add code that is not supported in all versions of PHP that we support, such as trailing commas in function parameters:

```
$this->assertEquals( 100, $number, );
```

- Verify that your code editor (if configured properly) catches this violation
